docs/how-to-deploy-and-call-a-contract/how-to-deploy-and-call-a-contract.md

```diff
@@ -232,6 +232,54 @@ The major differences between here and [local tests](../how-to-test-a-contract.m
 1. the contract needs to be deployed first;
 2. the contract instance is connected to a real provider, which broadcasts transactions to the blockchain.
 
+### next ()
```

Comment:
Delete parentheses here, since we're not describing the next function. Also make it a h4 (####).